raven-os/essentials/0.1.0/manifest.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""
+Build manifest for the `essential` and `essential-dev` package.
+They are virtual packages over the bare-bones essentials packages needed for
+a bootable and stable system.
+"""
+
+from textwrap import dedent
+from nbuild.stdenv.package import package
+from nbuild.stdenv.build import current_build
+
+
+@package(
+    id=f'stable::raven-os/essentials#0.1.0',
+    description='Core packages needed for a bootable and stable system.',
+    run_dependencies={
+        'stable::kernel/linux': '*',
+        'stable::raven-os/corefs': '*',
+        # TODO -- Add essentials packages here
+    },
+)
+def build_essentials():
+    package = current_build().current_package
+
+    # Hacky way of making a virtual package until nest and nbuild fully
+    # support them
+    open(f'{package.install_dir}/.essentials.nestkeep', 'w+')
```

Comment:
Garbage collection will probably close this at the end of the function, however you should still close it manually.
Also, why `w+` ? It's like `w` but with read permission as well, `w` already creates the file if it doesn't exist.
```suggestion
    open(f'{package.install_dir}/.essentials.nestkeep', 'w').close()
```
(Didn't notice it before but it also applies to `nbuild/stdenv/install.py:14` from the `nbuild` repository)

Reply:
Nice catch